src/Libraries/RevitNodes/RevitNodes.csproj

```diff
@@ -45,6 +45,9 @@
       <HintPath>$(DYNAMOAPI)\DynamoServices.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="GeoTimeZone">
+      <HintPath>..\..\packages\GeoTimeZone.1.1.0\lib\portable-net40+sl50+win+wp80\GeoTimeZone.dll</HintPath>
```

Comment:
if we are using this package, we should add license information to the repo:
https://raw.githubusercontent.com/mj1856/GeoTimeZone/master/LICENSE